src/main/java/net/javadiscord/javabot/systems/help/AutoCodeFormatter.java

```diff
@@ -106,8 +106,8 @@ void handleMessageEvent(@Nonnull MessageReceivedEvent event, boolean isFirstMess
 		}
 
 		if (isFirstMessage || !event.getMessage().getMentions().getUsers().isEmpty() ||
-				!event.getMessage().getMentions().getRoles().isEmpty() ||
```

Comment:
I think this `!` was correct.

Reply:
I've noticed that and put it back in https://github.com/Java-Discord/JavaBot/pull/425/commits/22550ec6b4351f0d9f57ee5a750844f8f99ea1e0